Atomic/trust.py

```diff
@@ -0,0 +1,173 @@
+from . import util
+from . import Atomic
+import os
+import argparse
+import json
+import yaml
+
+ATOMIC_CONFIG = util.get_atomic_config()
+
+def cli(subparser):
+    # atomic trust
+    pubkeys_dir = util.get_atomic_config_item(['pubkeys_dir'], atomic_config=ATOMIC_CONFIG)
+
+    trustp = subparser.add_parser("trust",
+                                 description="Manage trust policy for registries")
+    commonp = argparse.ArgumentParser(add_help=False)
+    registry_help="""Registry to manage trust policy for, REGISTRY[/REPOSITORY].
+                          Trust policy allows for nested scope.
+                          Example: registry.example.com defines trust for entire registry.
+                          Example: registry.example.com/acme defines trust for specific repository."""
+    sigstore_help="""Signature server type (default: docker)
+                     dir: local file 
+                     docker: static web server 
+                     atomic: openshift-based atomic registry"""
+    commonp.add_argument("--pubkeys", dest="pubkeys", nargs='*',
+                         help=_("Absolute path of installed public key(s) to trust for TARGET. "
+                                "May be a list of multiple trusted public keys. "
+                                "File(s) must exist before using this command. "
+                                "Default directory is %s" % pubkeys_dir))
+    commonp.add_argument("--sigstoretype", dest="sigstoretype", default="docker",
+                         choices=['dir', 'docker', 'atomic'],
+                         help=sigstore_help)
+    commonp.add_argument("--type", dest="trust_type", default="signedBy",
+                         choices=['signedBy', 'insecureAcceptAnything', 'reject'],
+                         help="Trust type (default: signedBy)")
+    commonp.add_argument("--keytype", dest="keytype", default="GPGKeys",
+                         help="Public key type (default: GPGKeys)")
+    commonp.add_argument("--sigstore", dest="sigstore",
+                         help=_("URL and path of remote signature server, "
+                                "https://sigstore.example.com/signatures. "
+                                "Ignored with 'atomic' sigstoretype."))
+    commonp.add_argument("registry", 
+                         help=registry_help)
+    subparsers = trustp.add_subparsers()
+    addp = subparsers.add_parser("add", parents=[commonp],
+                                 help="Add a new trust policy for a registry")
+    addp.set_defaults(_class=Trust, func="add")
+    removep = subparsers.add_parser("remove",
```

Comment:
I like "remove" but it's probably best to be consistent with the other args in the tool.